library/std/src/os/fortanix_sgx/ffi.rs

```diff
@@ -34,5 +34,68 @@
 
 #![unstable(feature = "sgx_platform", issue = "56975")]
 
-#[unstable(feature = "sgx_platform", issue = "56975")]
```

Comment:
Even though the re-export was unstable, that didn't change the stability of the exported items. So marking the trait and implementation below as stable is correct (or at least consistent with how it was before).